examples/step-61/doc/intro.dox

```diff
@@ -139,186 +139,330 @@ that is only defined in cell interiors. Consequently,
-We use shape functions defined on spaces FE_DGQ and FE_FaceQ to 
-approximate pressures, i.e., $p_h = \sum a_i \phi_i,$ 
-where $\phi_i$ are shape functions of FESystem. 
-We construct the local system by using discrete weak gradients of 
-shape functions of FE_DGQ and FE_FaceQ.
-The discrete weak gradients of shape functions $\nabla_{w,d} \phi$ are defined as 
-$\nabla_{w,d} \phi = \sum_{i=1}^m c_i \mathbf{w}_i,$ 
-where $\mathbf{w}_i$ is the basis function of $RT(k)$. 
-
-Using integration by parts, we have a small linear system 
-on each element $T$,
+
+<h3> Representing the weak gradient </h3>
+
+Since $p_h$ is an element of a finite element space, we can expand it in a basis
+as we always do, i.e., we can write
+@f{equation*}{
+  p_h(\mathbf x) = \sum_j P_j \varphi_j(\mathbf x).
+@f} 
+Here, since $p_h$ has two components (the interior and the interface components),
+the same must hold true for the basis functions $\varphi_j(\mathbf x)$, which we
+can write as $\varphi_j = (\varphi_j^\circ,\varphi_j^\partial)$. If you've
+followed the descriptions in step-8, step-20, and the
+@ref vector_valued "documentation module on vector-valued problems",
+it will be no surprise that for some values of $j$, $\varphi_j^\circ$ will be
+zero, whereas for other values of $j$, $\varphi_j^\partial$ will be zero -- i.e.,
+shape functions will be of either one or the other kind. That is not important,
+here, however. What is important is that we need to wonder how we can represent
+$\nabla_{w,d} \varphi_j$ because that is clearly what will appear in the
+problem when we want to implement the bilinear form 
 @f{equation*}
-\int_{T} \left(\nabla_{w,d} \phi \right) \cdot \mathbf{w} \mathrm{d}x= 
-\int_{T^\partial} \phi^{\partial} \left(\mathbf{w} \cdot \mathbf{n}\right) \mathrm{d}x- 
-\int_{T^\circ} \phi^{\circ} \left(\nabla \cdot \mathbf{w}\right) \mathrm{d}x,  
-\quad \forall \mathbf{w} \in RT_{[k]}(E),
+\mathcal{A}_h\left(p_h,q_h\right)
+  = \sum_{K \in \mathbb{T}}
+    \int_K \mathbf{K} \nabla_{w,d} p_h \cdot \nabla_{w,d} q_h \;\mathrm{d}x,
 @f}
 
+The key point is that $\nabla_{w,d} \varphi_j$ is known to be a member of the
+"broken" Raviart-Thomas space $DGRT_s$. What this means is that we can
+represent (on each cell $K$ separately)
 @f{equation*}
-\sum_{i=1}^m c_i \int_T \mathbf{w}_i \cdot \mathbf{w}_j \mathrm{d}x = 
-\int_{T^{\partial}} \phi_i^{\partial} 
-\left(\mathbf{w}_j \cdot \mathbf{n} \right) \mathrm{d}x - 
-\int_{T^{\circ}} \phi_i^{\circ} \left (\nabla \cdot \mathbf{w}_j \right)\mathrm{d}x,
+\nabla_{w,d} \varphi_j|_K
+  = \sum_k C_{jk}^K \mathbf v_k|_K
 @f}
-which can be simplified to be
-@f{equation*}
-\mathbf{C}_{E}\mathbf{M}_{E} = \mathbf{F}_{E},
+where the functions $\mathbf v_k \in DGRT_s$, and where $C^K$ is a matrix of
+dimension 
+@f{multline*}{
+ \text{dim}\left(DGQ_k(K) \times DGQ_r(K)\right) \times \text{dim}\left(RT_s(K)\right)
+  \\
+ =
+ \left(\text{dim}(DGQ_k(K)) + \text{dim}(DGQ_r(K))\right) \times \text{dim}\left(RT_s(K)\right).
```

Comment:
The line break doesn't look good unless the screen used for rendering is very small. I think it would be slightly better to get rid of the line break.

Another possibility would be to write
```html
where the functions $\mathbf v_k \in DGRT_s$, and where $C^K$ is a matrix of
dimension
@f{equation*}{
 \left(\text{dim}(DGQ_k(K)) + \text{dim}(DGQ_r(K))\right) \times \text{dim}\left(RT_s(K)\right)
@f}
as $\text{dim}\left(DGQ_k(K) \times DGQ_r(K)\right) = \left(\text{dim}(DGQ_k(K)) + \text{dim}(DGQ_r(K))\right)$.
```

Reply:
Yes, this is a general problem with the `{multline}` environment -- doxygen asks latex to render this with an extremely wide page width. I've used `align` instead.

Reply:
This looks much better. :+1: